### PR TITLE
Updated installation steps and troubleshooting for dev environment setup of cordova-docs

### PR DESCRIPTION
**Pull request by gandhirajan**
This PR contains updated installation steps for development environment setup of cordova-docs. Also troubleshooting section is added under installation page. Please note that the installation related troubleshooting instructions are moved from general troubleshooting page to installation page

### CODE REVIEW

File: doc/installing-a-development-environment.md [hunk 1]
```diff
@@ -131,6 +135,31 @@ Verify your make installation by running:
 
     make --version
 
+
+### Troubleshooting
+
+#### SSL certificate issue while executing 'gem install bundler' command in Windows
+
+Try copying the certificate from [GlobalSignRootCerficateAuthority] into `C:\Ruby22\lib\ruby\2.2.0\rubygems\ssl_certs` folder
```
Comment (janpio) [inline doc/installing-a-development-environment.md]: This only works for this specific version of Ruby and installation path.
I don't think I had to do that when I installed on Windows.
Is there a public link that gives the same advice one can link to?

File: doc/installing-a-development-environment.md [hunk 2]
```diff
@@ -90,11 +90,15 @@ Verify your Node.js installation by running:
     node --version
     npm --version
 
+### Local repo setup
+
+Clone the [cordova-docs] github repo to the local folder
```
Comment (janpio) [inline doc/installing-a-development-environment.md]: 1. github => GitHub
1. to a specific folder or a random folder one likes?
1. Misses dot at end of sentence.

File: doc/installing-a-development-environment.md (hunk 1 above)
Comment (gandhirajan) [reply doc/installing-a-development-environment.md]: Hi Jan, SSL certificate issue is specific to windows. Also the ruby version and path are inline with the installation steps mentioned in the doc.

File: doc/installing-a-development-environment.md (hunk 2 above)
Comment (gandhirajan) [reply doc/installing-a-development-environment.md]: @janpio Hi Jan, the review comments are implemented and thanks for your time.

File: doc/installing-a-development-environment.md [hunk 3]
```diff
@@ -131,6 +135,31 @@ Verify your make installation by running:
 
     make --version
 
+
+### Troubleshooting
+
+#### SSL certificate issue while executing 'gem install bundler' command in Windows
```
Comment (janpio) [inline doc/installing-a-development-environment.md]: in Windows => on Windows

Comment (janpio) [reply doc/installing-a-development-environment.md]: ' => ` so the command is recognized as a command

Comment (janpio) [reply doc/installing-a-development-environment.md]: "command" can then probably also be removed

**Pull request merged**